contact_detection_plugin:
  plugin: "ContactDetectionPlugin"
  # Subscriber and Publisher topic names
  # LF: Left Front leg, RF: Right Front leg, LH: Left Hind leg, RH: Right Hind leg
  wrench_lf_topic: "/state_estimator/contact_force_lf_foot"
  wrench_rf_topic: "/state_estimator/contact_force_rf_foot"
  wrench_lh_topic: "/state_estimator/contact_force_lh_foot"
  wrench_rh_topic: "/state_estimator/contact_force_rh_foot"

  pub_topic: "contact_detection"

  # thresholds for contact detection
  grf_threshold: 30.0